lib/reek/context/attribute_context.rb

```diff
@@ -15,6 +15,15 @@ def full_comment
         send_expression.full_comment || ''
       end
 
+      def instance_method?
+        true
+      end
+
+      # Was this method defined with an instance method-like syntax?
```

Comment:
This is overwritten depending on the context, right? Can we somehow document in one place what we mean with "instance-like syntax" and then drop this comment here?